modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PythonClientCodegen.java

```diff
@@ -17,48 +16,57 @@
 
 package org.openapitools.codegen.languages;
 
-import io.swagger.v3.oas.models.examples.Example;
+import io.swagger.v3.core.util.Json;
+import io.swagger.v3.oas.models.media.*;
 import io.swagger.v3.oas.models.media.ArraySchema;
+import io.swagger.v3.oas.models.media.MediaType;
 import io.swagger.v3.oas.models.media.Schema;
 import io.swagger.v3.oas.models.parameters.Parameter;
-import org.apache.commons.io.FilenameUtils;
+import io.swagger.v3.oas.models.parameters.RequestBody;
+import io.swagger.v3.oas.models.security.SecurityScheme;
 import org.apache.commons.lang3.StringUtils;
 import org.openapitools.codegen.*;
+import org.openapitools.codegen.CodegenDiscriminator.MappedModel;
 import org.openapitools.codegen.meta.features.*;
 import org.openapitools.codegen.utils.ModelUtils;
+import org.openapitools.codegen.utils.ProcessUtils;
+import org.openapitools.codegen.meta.GeneratorMetadata;
+import org.openapitools.codegen.meta.Stability;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import com.github.curiousoddman.rgxgen.RgxGen;
 
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+import java.time.format.DateTimeFormatter;
 import java.io.File;
 import java.util.*;
 import java.util.regex.Pattern;
+import java.util.regex.Matcher;
 
-import static org.openapitools.codegen.utils.StringUtils.camelize;
-import static org.openapitools.codegen.utils.StringUtils.underscore;
+import static org.openapitools.codegen.utils.OnceLogger.once;
 
-public class PythonClientCodegen extends DefaultCodegen implements CodegenConfig {
+public class PythonClientCodegen extends PythonLegacyClientCodegen {
```

Comment:
FYI. My previous approach (PR) completely separated `python` and `python-legacy` so that PythonClientCodegen extends DefaultCodegen instead of PythonLegacyClientCodegen. I guess we can do the decoupling later with another PR instead.

Reply:
Let's do that later. Doing this a step at a time reduces the risk that we will miss something. As it is right now this PR is huge. That future refactor will be on 2 or 3 files and will be easier to understand then.